.github/workflows/schedule-last-call-date-end.yml

```diff
@@ -0,0 +1,162 @@
+name: Schedule Status Update Based on Last Call Time
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - main
+
+jobs:
+  check-merged-file:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
```

Comment:
Use the self-hosted runner:

`runs-on: [self-hosted, Linux, medium, ephemeral]`